test/sagemaker_tests/pytorch/training/conftest.py

```diff
@@ -30,7 +30,7 @@
 from . import get_efa_test_instance_type
 
 from .utils import get_ecr_registry, NightlyFeatureLabel, is_nightly_context
-
+from test.test_utils import get_framework_and_version_from_tag, get_cuda_version_from_tag
```

Comment:
I suspect this may break SM Local tests because they don't put `test.test_utils` on the test PYTHONPATH. Please try running the tests again.

You can get `get_framework_and_version_from_tag` from https://github.com/aws/deep-learning-containers/blob/master/test/sagemaker_tests/pytorch/training/integration/__init__.py.

You could perhaps copy `get_cuda_version_from_tag` as well, from `test/test_utils` into that `integration/__init__.py`.